cdap-common/src/main/java/co/cask/cdap/common/kerberos/NoOpOwnerAdmin.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.common.kerberos;
+
+import co.cask.cdap.common.AlreadyExistsException;
+import co.cask.cdap.proto.id.KerberosPrincipalId;
+import co.cask.cdap.proto.id.NamespacedEntityId;
+
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+/**
+ * No-op implementation of {@link OwnerAdmin}.
+ */
+public class NoOpOwnerAdmin implements OwnerAdmin {
```

Comment:
So this binding is used only at place where we don't support Kerberos and Owner. For example in Preview/Standalone.
I do see what you mean though. Ideally, AppFabric should only be expecting Owner and dealing with it if Kerberos is enabled (which is not the case in preview/standalone etc). Though, we don't we have that check in place as of now. I will add that check so that it kicks in only if Kerberos is enabled.